test/dynamo/test_functions.py

```diff
@@ -1741,21 +1741,29 @@ def fn(x):
                 x = torch.randn(10)
                 self.assertEqual(opt_fn(x), fn(x))
 
-    def test_pos(self):
-        def fn(x, y):
-            return operator.pos(x) * operator.pos(y)
+    def test_unary_functions(self):
```

Comment:
Since you are not testing all possible unary functions, it might be better to just define test_abs, test_truth etc. Having separate tests also make repro etc easier.

Reply:
done